### ISSUE TITLE
Update the onnx-mlir version

### ISSUE THREAD

**Issue by uditagarwal97** [DependableSystemsLab/LLTFI#10]
It looks like ONNX-MLIR is now a lot more mature and supports 116 ML models out of 128 Models from the ONNX Zoo, including large models like Bert and GPT. Ref: [https://github.com/onnx/onnx-mlir/issues/128]( https://github.com/onnx/onnx-mlir/issues/128).

We should seriously consider upgrading the _onnx-mlir_ version being used with LLTFI.

**Comment by uditagarwal97** [comment]
@abrahamchan I have updated the onnx-mlir version to commit id `9c62e1a4`. However, I had to make certain changes in instrumentation passes of the onnx-mlir code to get LLTFI working with it.  We have to decide whether 1) I should keep my changes as a patch and apply the patch to onnx-mlir? Or 2) we fork the onnx-mlir repository to DependableSystemsLab GitHub org and I make changes to the forked version of onnx-mlir.
